### PR TITLE
Feature/allow opening pre parsed xml files

### PR DESCRIPTION
**Pull request by marco-rabozzi**
Add support for loading/opening pre-parsed arxml file with the xml.etree.ElementTree library

### CODE REVIEW

File: autosar/workspace.py [hunk 1]
```diff
@@ -174,16 +174,32 @@ def openXML(self,filename):
         if self.packageParser is None:
             self.packageParser = autosar.parser.package_parser.PackageParser(self.version)
         self._registerDefaultElementParsers(self.packageParser)
+    
+    def openXML(self, filename):
+        xmlroot = parseXMLFile(filename)
+        self.openXML(xmlroot)
```
Comment (andredami) [inline autosar/workspace.py:180]: I think this should be
```suggestion
        self._openXML(xmlroot)
```

**Review by andredami — CHANGES_REQUESTED** [review]
Minor bug

**Comment by marco-rabozzi** [comment]
[[hux-autosar-sil] fast loading and merging of arxml files](https://trello.com/c/pQp1NaTg/812-hux-autosar-sil-fast-loading-and-merging-of-arxml-files)

**Review by andredami — APPROVED** [review]
LGTM

**Pull request merged**